src/main/resources/entity-types/inventory/simple_instance_type.json5

```diff
@@ -0,0 +1,138 @@
+{
+  id: 'af44e2e0-12e0-4eec-b80d-49feb33a866c',
+  name: 'simple_instance_type',
+  private: true,
+  sources: [
+    {
+      type: 'db',
+      alias: 'instance_type',
+      target: 'src_inventory_instance_type',
+    },
+  ],
+  requiredPermissions: [
+    'inventory-storage.instance-types.collection.get',
+    'inventory-storage.instance-types.item.get'
+  ],
+  columns: [
+    {
+      name: 'id',
+      sourceAlias: 'instance_type',
+      dataType: {
+        dataType: 'rangedUUIDType',
+      },
+      isIdColumn: true,
+      queryable: true,
+      visibleByDefault: false,
+      valueGetter: ":sourceAlias.jsonb->>'id'",
```

Comment:
would we be able to just use :sourceAlias.id here?

Reply:
Nice catch! That would make it use an index, too!